content/docs/setup/kubernetes/additional-setup/config-profiles/index.md

```diff
@@ -53,12 +53,17 @@ The components marked as **X** are installed within each profile:
 Some profiles have an authentication variant, with `-auth` appended to the name, which adds the following
 security features to the profile:
 
-| | default | demo | minimal | sds |
+| | default | demo | minimal | sds-auth |
 | --- | --- | --- | --- | --- |
 | Control Plane Security | | X | | |
 | Strict Mutual TLS | | X | | X |
 | SDS | | | | X |
 
+{{< tip >}}
+Control plan security still requires the use of secret volume mounts, and is therefore
+incompatible with SDS.
```

Comment:
controlplane sds is WIP (there isn't technical blocker but just timeline) 
probably tweak somethinlg like ```enable sds on controlplane is work in progress``` ?